src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size() 메소드를 이용해 Set의 크기 확인")
+    void setSize() {
+        int size = numbers.size();
+
+        assertThat(size)
+                .isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("Set의 contains() 를 이용해 1,2,3의 값이 존재하는지 확인")
+    void setContainsAlwaysTrue(int index) {
+        assertThat(numbers.contains(index)).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    @DisplayName("Set의 contains() 를 이용해 포함되어있을때와 미포함되어있을떄를 확인")
+    void setContains(int index, boolean isContains) {
```

Comment:
default delimiter를 사용해도 좋을 것 같습니다 🙂
```suggestion
    @CsvSource(value = {"1,true", "2,true", "3,true", "4,false", "5,false"})
```